### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by ZJH2233** [changhongjian/Deep3DFaceReconstruction-pytorch#8]
Hi，Thanks for your project.
I have read the paper about the project.The loss function is complicated.
So,did you write the loss function?
Or,if can't implement the loss function,What should I do to fine-tuning or transfer learning?
(我看注释里有中文，用中文直白一点的问问。原论文损失函数太复杂了我实现不了请问您实现了吗，如果实现不了损失函数又想调整训练参数适应我的数据集该怎么做？我之前是想迁移学习或者微调但是这两个我以前没做过好像是需要损失函数的)

**Comment by ZJH2233** [comment]
In this Repo [https://github.com/yuhaoooo/Deep3DFaceModel-Pytorch](url)

> so you can use Deep3DFaceReconstruction-pytorch repo to train the network. And the pretrained model is also from this repo.

How do I train the network?

**Comment by ForrestPi** [comment]
@ZJH2233  可以参考一下 https://github.com/zhipeng-fan/3D-face
